feconf.py

```diff
@@ -257,6 +257,9 @@ def get_empty_ratings():
 BULK_EMAIL_INTENT_LEARNER_REENGAGEMENT = 'bulk_email_learner_reengagement'
 BULK_EMAIL_INTENT_TEST = 'bulk_email_test'
 
+MESSAGE_TYPE_SUGGESTION = "suggestion"
```

Comment:
Use single quotes in backend.

Swap order of this and the next line (try and keep things alphabetical if there's no real reason to do otherwise).